skrub/_joiner.py

```diff
@@ -110,20 +111,20 @@ class FeatureAugmenter(TransformerMixin, BaseEstimator):
                       (aux_table_2, "Country name"),
                       (aux_table_3, "Countries")]
 
-    >>> fa = FeatureAugmenter(tables=aux_tables, main_key='Country')
+    >>> joiner = Joiner(tables=aux_tables, main_key='Country')
 
-    >>> augmented_table = fa.fit_transform(X)
+    >>> augmented_table = joiner.fit_transform(X)
     >>> augmented_table
         Country Country_aux  Population Country name  GDP (billion) Countries Capital
     0   France      France    68000000  French Republic       2937    France   Paris
-    1  Germany     Germany    84000000      Germany           4223   Germany  Berlin
-    2    Italy       Italy    59000000        Italy           2099    Italia    Rome
+    1   Germany     Germany   84000000      Germany           4223   Germany   Berlin
+    2    Italy       Italy    59000000        Italy           2099    Italia   Rome
```

Comment:
Maybe a small example of something that looks like a spatial join here, so that people find out that it is possible.